### DISCUSSION TITLE
Whisper.wasm on freepodcasttranscription.com

### DISCUSSION THREAD

**Discussion by lele85** [ggml-org/whisper.cpp#593] [Show and tell]
Hi there, i'm Emanuele Rampichini, Head of Engineering at Spreaker. We just launched a small free tool that i think can be useful for podcasters using this amazing project:

[https://freepodcasttranscription.com/](https://freepodcasttranscription.com/)

![freepodcasttranscription com_](https://user-images.githubusercontent.com/49144/224026683-317676c3-414c-4f63-b197-f6f2deda96a0.png)

The srt can then be added on the podcast RSS feed, on any hosting platform the user is happy with following the Podcasting 2.0 specifications.

**Comment by lele85** [comment]
If you have any feature requests let me know here and i will try and implement them :)

**Comment by patrikzudel** [comment]
Amazing! @lele85 

Could you add the option to download the .srt without publishing it? 

Thank you!

// Edit: I noticed that this is actually possible when going into “Your transcriptions” even before publishing. Still, I think a button on the main page would be nice to have.
